### DISCUSSION TITLE
Meta-discussion: what this place is for

### DISCUSSION THREAD

**Discussion by 2colours** [Raku/problem-solving#359] [General]
Hi there,

I generally like the idea of GH Discussions as opposed to opening issues for everything; kind of like an inbetween for issues and real-time chat. I noticed that there is a Discussions tab for this repository as well so first off I'd try to find a purpose for it - in no other form than a discussion! :)

When this repository was created as kind of an evolutionary step of the language development workflow, creating issues here suddenly became a common sight but also quite a heavyweight way to work on very diverse kind of issues around Raku. Over time, the influence of these issues got lower and lower - at the same time, much of the tension of opening an issue remained. These issues never stopped to indicate "we need to do something about this" but they got less and less likely to be resolved.

I think Discussions can somewhat lower the carried emotional baggage of a "problem-solving issue".

I tried to summarize my proposal for a principle. **What I came up with: a topic has a better place here than among issues if:**
- **there is no well-understood and highly consensual "issue"**
This seems fairly straightforward: the format of a forum is more fitting for something that might not be an "issue" in the first place. Also it's less pressurizing and threatening than an issue.
- **the analysis of a certain topic is more important than to get a solution**
This kind of overlaps with the previous point but say, even if there is a well-understood issue, it might be nearly impossible to just outright resolve it, in which case again, it's kind of a nuisance to mix it with things that are well-understood and needing a solution. The analysis of the situation can still be highly useful.

For the formality as well, I think these discussions can be somewhat more convenient and looser than issues - which, in turn, might be more worth keeping clean.

For the time being, I'll try to use this page the way I described but ideas are welcome. ^^

**Comment by raiph** [comment]
As I understand it "problem solving issues" were meant to be for, or at least turned out to only be good for, things that were already acknowledged by a large swathe of the community to be a complex problem that needed resolution. As such they should have been, or should be, rarely opened, and opened when discussion has recently lit up (note, lit as in light, not heat, definitely not heat), and should be active until resolved, or at least until a plan drops into place and is being executed.

What I think happened is that folk instead started filing issues for much lesser problems. I even did this myself, ignorant of what they were good for.
